frontend/pages/hosts/ManageHostsPage/components/LinkCell/LinkCell.tsx

```diff
@@ -1,23 +1,27 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { push } from 'react-router-redux';
 
-import hostInterface from 'interfaces/host';
-import helpers from 'kolide/helpers';
-import PATHS from 'router/paths';
-import Button from 'components/buttons/Button';
+import { IHost } from '../../../../../interfaces/host';
```

Comment:
Is there a config we can change so that we don't need to use the `../../..` for these imports?